src/titiler/mosaic/titiler/mosaic/factory.py

```diff
@@ -237,11 +245,22 @@ def info_geojson(
                     reader_options=reader_params.as_dict(),
                     **backend_params.as_dict(),
                 ) as src_dst:
-                    info = src_dst.info()
+                    bounds = src_dst.get_geographic_bounds(crs or WGS84_CRS)
+                    if bounds[0] > bounds[2]:
+                        pl = Polygon.from_bounds(-180, bounds[1], bounds[2], bounds[3])
```

Comment:
wondering how safe this is to assume, if dealing with a pyproj CRS and the area of use is defined that could be one way to get this, but I wonder for other codes and coordinate system axis orders... Probably something though that isn't super important and can be solved later

Reply:
oh wait you get the bounds first, okay then this is porbably fine as xy order is enforced

Reply:
👍 good point, let's wait until someone gets the issue 😅 

For now it will cover 98% of the users I think